test/spec/runners/ssz_static.ex

```diff
@@ -5,43 +5,31 @@ defmodule SSZStaticTestRunner do
   Runner for SSZ test cases. `run_test_case/1` is the main entrypoint.
   """
 
-  @enabled [
```

Comment:
adding disabled tests is better than enabled. For example there was a typo: "AttestorSlashing". Also it gives us an overview to which types are missing

Reply:
I agree. I didn't do it initially since the unimplemented types were a lot (in fact, this list doesn't include all of them).